framework/CodeInterfaceBaseClass.py

```diff
@@ -43,7 +43,25 @@ def __init__(self):
       @ In, None
       @ Out, None
     """
-    self.inputExtensions = []
+    self.inputExtensions = []            # list of input extensions
+    self.ignoreInputExtensions = ['zip'] # the list of input extensions that will be ignored by the code interface.
```

Comment:
what does it mean?  Why is it in the base class? It should be in the derived class

Reply:
removed to Saphire Code interface. It is needed, since the input file with '.zip' is required, but it will not be perturbed.